### DISCUSSION TITLE
The Ultimate Explanation to Integrate Iro Picker in IONIC in 2021

### DISCUSSION THREAD

**Discussion by pedros89** [jaames/iro.js#183] [General]
Hi James,

I finally managed to integrate your color picker in IONIC thanks to the help of Vikas who made this great video on how one can do it:

https://youtu.be/g4rQBQDUSJY

The typical application would be then to send the data via Bluetooth to an Arduino to remotely control LED RGB lamps. I am working to make this stuff also available open-sorce.
Could you share this video on your GitHub main page and on your website? this is exactly what many IONIC developers are looking for.
I worked a lot to find out a solution and finally it is there.
You can see the post in which I was struggling with the solution here:
https://github.com/jaames/iro.js/issues/139
Now everything is sorted.
Happy new year,
Pietro

**Comment by jaames** [comment]
Cool find! I'll look into fitting this into the documentation with some other framework/library examples soon :)
